api/base.rb

```diff
@@ -0,0 +1,21 @@
+require 'grape'
+require_relative '../tools/integer'
+
+module API
+  class Base < Grape::API
+    format :json
+    prefix :api
+
+    resource :ping do
+      get do
+        { ping: 'pong' }
+      end
+    end
+
+    resource :dice do
+      get do
+        { result: rand(1..params[:size] || 6).to_roman }
```

Comment:
#genio on the roman.

However, I'd make the argument like die_roll(faces: 6) so we can randomize also 1..100 if needed.